docs/docsite/rst/network/user_guide/faq.rst

```diff
@@ -0,0 +1,34 @@
+.. _network_faq:
+
+*******************
+Ansible Network FAQ
+*******************
+
+How can I improve performance for network playbooks?
+====================================================
+
+* Set ``strategy: free`` at the playbook level if you are running on multiple hosts
```

Comment:
Wonder if each of the "hows" should be separate headings with anchors. Would improve formatting and give us the ability to link directly to the relevant section

Looking at the rendered version the current bullet point version looks a bit strange to me